src/app/features/tech-record/components/vehicle-technical-record/vehicle-technical-record.component.ts

```diff
@@ -82,9 +82,17 @@ export class VehicleTechnicalRecordComponent implements OnInit, AfterViewInit {
 
   handleFormState() {
     if (this.isEditing) {
+      const isPsv = this.vehicleTechRecord?.techRecord.find(record => record.statusCode === StatusCodes.CURRENT)?.vehicleType === VehicleTypes.PSV;
```

Comment:
We've got this code in a few places now, we should look to move vehicle type checking into the service, as its consumed in atleast 3 places. Not one for now but for everyone to be aware of.

Reply:
Sure, thanks for the heads up.